content/docs/user-guide/engine/_index.md

```diff
@@ -1,14 +1,34 @@
 ---
-description: The following topics will guide you through programming with and for O3DE. 
+description: Learn about Open 3D Engine's core systems and how you can extend the systems by programming.
```

Comment:
```suggestion
description: Learn about the core systems of Open 3D Engine (O3DE) and how you can extend those systems by programming.
```
Add parenthetical "(O3DE)", per our [usual usage in metadata descriptions](https://docs.o3de.org/docs/contributing/to-docs/style-guide/metadata/).